404.html

```diff
@@ -14,7 +14,6 @@
     <link rel="stylesheet" href="./css/style.css" />
     <title>Portfolio of Farhan(Hantasmagoria)</title>
     <script>
-      document.addEventListener("click", musicPlay);
```

Comment:
leaving this here disrupted the order in which the event listener could access the "musicPlay" function. moved this down to right below the function called.